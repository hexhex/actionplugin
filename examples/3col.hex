node(X) :- edge(X,Y).
node(Y) :- edge(X,Y).

colored(X,r) v colored(X,g) v colored(X,b) :- node(X).

:- edge(X,Y), colored(X,C), colored(Y,C).

edge( 2, 4). edge( 2, 3). edge( 3, 5).
edge( 4, 6). edge( 4, 5). edge( 5, 7).
             edge( 6, 7).
