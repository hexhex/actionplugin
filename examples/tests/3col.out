{colored(2,b), colored(3,g), colored(4,g), colored(5,b), colored(6,b), colored(7,g), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,b), colored(3,g), colored(4,g), colored(5,b), colored(6,b), colored(7,r), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,b), colored(3,g), colored(4,g), colored(5,b), colored(6,r), colored(7,g), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,b), colored(3,g), colored(4,g), colored(5,r), colored(6,b), colored(7,g), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,b), colored(3,g), colored(4,g), colored(5,r), colored(6,r), colored(7,b), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,b), colored(3,g), colored(4,g), colored(5,r), colored(6,r), colored(7,g), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,b), colored(3,g), colored(4,r), colored(5,b), colored(6,b), colored(7,g), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,b), colored(3,g), colored(4,r), colored(5,b), colored(6,b), colored(7,r), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,b), colored(3,g), colored(4,r), colored(5,b), colored(6,g), colored(7,r), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,b), colored(3,r), colored(4,g), colored(5,b), colored(6,b), colored(7,g), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,b), colored(3,r), colored(4,g), colored(5,b), colored(6,b), colored(7,r), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,b), colored(3,r), colored(4,g), colored(5,b), colored(6,r), colored(7,g), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,b), colored(3,r), colored(4,r), colored(5,b), colored(6,b), colored(7,g), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,b), colored(3,r), colored(4,r), colored(5,b), colored(6,b), colored(7,r), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,b), colored(3,r), colored(4,r), colored(5,b), colored(6,g), colored(7,r), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,b), colored(3,r), colored(4,r), colored(5,g), colored(6,b), colored(7,r), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,b), colored(3,r), colored(4,r), colored(5,g), colored(6,g), colored(7,b), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,b), colored(3,r), colored(4,r), colored(5,g), colored(6,g), colored(7,r), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,g), colored(3,b), colored(4,b), colored(5,g), colored(6,g), colored(7,b), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,g), colored(3,b), colored(4,b), colored(5,g), colored(6,g), colored(7,r), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,g), colored(3,b), colored(4,b), colored(5,g), colored(6,r), colored(7,b), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,g), colored(3,b), colored(4,b), colored(5,r), colored(6,g), colored(7,b), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,g), colored(3,b), colored(4,b), colored(5,r), colored(6,r), colored(7,b), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,g), colored(3,b), colored(4,b), colored(5,r), colored(6,r), colored(7,g), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,g), colored(3,b), colored(4,r), colored(5,g), colored(6,b), colored(7,r), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,g), colored(3,b), colored(4,r), colored(5,g), colored(6,g), colored(7,b), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,g), colored(3,b), colored(4,r), colored(5,g), colored(6,g), colored(7,r), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,g), colored(3,r), colored(4,b), colored(5,g), colored(6,g), colored(7,b), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,g), colored(3,r), colored(4,b), colored(5,g), colored(6,g), colored(7,r), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,g), colored(3,r), colored(4,b), colored(5,g), colored(6,r), colored(7,b), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,g), colored(3,r), colored(4,r), colored(5,b), colored(6,b), colored(7,g), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,g), colored(3,r), colored(4,r), colored(5,b), colored(6,b), colored(7,r), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,g), colored(3,r), colored(4,r), colored(5,b), colored(6,g), colored(7,r), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,g), colored(3,r), colored(4,r), colored(5,g), colored(6,b), colored(7,r), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,g), colored(3,r), colored(4,r), colored(5,g), colored(6,g), colored(7,b), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,g), colored(3,r), colored(4,r), colored(5,g), colored(6,g), colored(7,r), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,r), colored(3,b), colored(4,b), colored(5,g), colored(6,g), colored(7,b), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,r), colored(3,b), colored(4,b), colored(5,g), colored(6,g), colored(7,r), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,r), colored(3,b), colored(4,b), colored(5,g), colored(6,r), colored(7,b), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,r), colored(3,b), colored(4,b), colored(5,r), colored(6,g), colored(7,b), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,r), colored(3,b), colored(4,b), colored(5,r), colored(6,r), colored(7,b), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,r), colored(3,b), colored(4,b), colored(5,r), colored(6,r), colored(7,g), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,r), colored(3,b), colored(4,g), colored(5,r), colored(6,b), colored(7,g), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,r), colored(3,b), colored(4,g), colored(5,r), colored(6,r), colored(7,b), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,r), colored(3,b), colored(4,g), colored(5,r), colored(6,r), colored(7,g), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,r), colored(3,g), colored(4,b), colored(5,r), colored(6,g), colored(7,b), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,r), colored(3,g), colored(4,b), colored(5,r), colored(6,r), colored(7,b), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,r), colored(3,g), colored(4,b), colored(5,r), colored(6,r), colored(7,g), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,r), colored(3,g), colored(4,g), colored(5,b), colored(6,b), colored(7,g), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,r), colored(3,g), colored(4,g), colored(5,b), colored(6,b), colored(7,r), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,r), colored(3,g), colored(4,g), colored(5,b), colored(6,r), colored(7,g), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,r), colored(3,g), colored(4,g), colored(5,r), colored(6,b), colored(7,g), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,r), colored(3,g), colored(4,g), colored(5,r), colored(6,r), colored(7,b), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
{colored(2,r), colored(3,g), colored(4,g), colored(5,r), colored(6,r), colored(7,g), edge(2,3), edge(2,4), edge(3,5), edge(4,5), edge(4,6), edge(5,7), edge(6,7), node(2), node(3), node(4), node(5), node(6), node(7)}
